src/Sylius/Component/Order/Factory/AdjustmentFactoryInterface.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Order\Factory;
+
+use Sylius\Component\Order\Model\AdjustmentInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+interface AdjustmentFactoryInterface extends FactoryInterface
```

Comment:
As for now, each custom factory interface already implemented, e.g. [Product's](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Product/Factory/ProductFactoryInterface.php), [Channel's](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Channel/Factory/ChannelFactoryInterface.php), [Attribute's](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Component/Attribute/Factory/AttributeFactoryInterface.php) etc., extends base factory interface.